### DISCUSSION TITLE
pyMOR Community Meeting 2023-05-09

### DISCUSSION THREAD

**Discussion by pmli** [pymor/pymor#2024] [Announcements]
The next pyMOR Community Meeting will take place Tuesday, May 9, at 14:00 CEST and 8:00 EDT, online at:

link: https://bbb.mpi-magdeburg.mpg.de/meeting/pymor
password: THaEo

Anyone interested in pyMOR can join.

Currently on the agenda are:
- [x] issue and PR discussion for 2023.1 release (@pmli)
- [x] NumFOCUS (@pmli)
- [x] packaging/ci infrastructure (@sdrave)
- [x] pyMOR School 2023 organization (@pmli)

Moderator: @pmli

**Comment by lbalicki** [comment]
Minutes

- NumFOCUS
  - pyMOR is now on NumFOCUS website (although link is not correct)
- pyMOR School 2023 organization
  - we are still waiting for room approval
  - get school homepage online now -> announce later
  - will contact some professor once AP is back
- packaging/ci infrastructure
  - new PR by SR
  - will remove hatch vcs stuff and manually add version
  - need to add version update to release check list
  - for dev setup probably best to install everything
- issue and PR discussion for 2023.1 release

Next meeting:  Friday 5/19, 14 CET, 8 EST
Moderator: Hendrik
